src/periodic_report_population.rs

```diff
@@ -0,0 +1,93 @@
+use crate::population_loader::{Age, CensusTract};
+
+use crate::Parameters;
+use ixa::{
+    context::{Context, ExecutionPhase},
+    create_report_trait, define_data_plugin,
+    error::IxaError,
+    global_properties::ContextGlobalPropertiesExt,
+    people::{ContextPeopleExt, PersonCreatedEvent},
+    report::{ContextReportExt, Report},
+};
+use serde::{Deserialize, Serialize};
+use std::path::{Path, PathBuf};
+
+use std::collections::HashSet;
+
+#[derive(Serialize, Deserialize, Clone, PartialEq)]
```

Comment:
Do you need to implement all of these? It looks like maybe not Deserialize, Clone, or PartialEq